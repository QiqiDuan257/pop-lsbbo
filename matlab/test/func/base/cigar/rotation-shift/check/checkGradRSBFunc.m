clear; clc;

% nohup matlab -nodesktop -nodisplay -nosplash <checkGradRSBFunc.m >checkGradRSBFunc.out 2>&1 &

funcName = 'popRSGradCigar_';

%%
popCheckCommGradRSBFunc(funcName);
