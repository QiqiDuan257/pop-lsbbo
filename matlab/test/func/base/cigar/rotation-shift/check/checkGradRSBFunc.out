
                            < M A T L A B (R) >
                  Copyright 1984-2016 The MathWorks, Inc.
                   R2016b (9.1.0.441655) 64-bit (glnxa64)
                             September 7, 2016

#############################################################################
#             $$$ Enjoy the <pop-lsbbo> pure-MATLAB Library $$$             #
#                        --- version: 0.00.000.1 ---                        #
#############################################################################
popRSGradCigar_: pass the test.
     3: runtime 1.04e-01.
popRSGradCigar_: pass the test.
     4: runtime 4.56e-03.
popRSGradCigar_: pass the test.
     5: runtime 3.30e-03.
popRSGradCigar_: pass the test.
     6: runtime 2.33e-03.
popRSGradCigar_: pass the test.
     7: runtime 1.55e-02.
popRSGradCigar_: pass the test.
     2: runtime 1.94e-03.
popRSGradCigar_: pass the test.
    10: runtime 1.93e-03.
popRSGradCigar_: pass the test.
   100: runtime 3.55e-02.
popRSGradCigar_: pass the test.
  1000: runtime 1.16e+00.
popRSGradCigar_: pass the test.
  2000: runtime 4.14e+00.
popRSGradCigar_: pass the test.
  4000: runtime 1.69e+01.
popRSGradCigar_: pass the test.
  6000: runtime 3.78e+01.
popRSGradCigar_: pass the test.
  8000: runtime 6.91e+01.
popRSGradCigar_: pass the test.
 10000: runtime 1.04e+02.
Total runtime: 2.33706e+02.
